.github/workflows/build-and-test.yml

```diff
@@ -0,0 +1,27 @@
+name: build-and-test
+
+on:
+  push:
+    # all branches
+  pull_request:
+    branches:
+      - main
+      - master
+
+  # This enables the Run Workflow button on the Actions tab.
+  workflow_dispatch:
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    steps:
+    
+      - uses: actions/checkout@v2
+
+      - uses: dylan-lang/install-opendylan@v1
```

Comment:
This should be @v2 now. I think v1 works but v2 adds support for macOS.  It's possible some of the ones I did originally may still be using @v1 as well.